docs/pages/web/sidenavigation.js

```diff
@@ -454,6 +459,53 @@ When building SideNavigation, we might want to render different combinations of
           />
         </MainSection.Subsection>
 
+        <MainSection.Subsection title="Collapsible" description="">
+          <MainSection.Card
+            cardSize="lg"
+            title="Items with icons"
+            sandpackExample={
+              <SandpackExample
+                code={collapsibleExample}
+                name="Collapsible with icons example"
+                previewHeight={500}
+              />
+            }
+          />
+          <MainSection.Card
+            cardSize="lg"
+            title="Items without icons"
+            sandpackExample={
+              <SandpackExample
+                code={collapsibleWithoutIconsExample}
+                name="Collapsible without icons example"
+                previewHeight={500}
+              />
+            }
+          />
+          <MainSection.Card
+            cardSize="lg"
+            title="Mixed items with and without icons"
+            sandpackExample={
+              <SandpackExample
+                code={collapsibleWithMixedIconsExample}
+                name="Collapsible mixed icons example"
+                previewHeight={500}
+              />
+            }
+          />
+          <MainSection.Card
+            cardSize="lg"
+            title="Collapsible header"
```

Comment:
nit I'd add some descriptions here to make it more relevant. For example, highlight the importance of onPreview for the Collapsible header example. 

Each example has nuances that might be worth highlighting. It can be a follow up PR.